packages/analytics/metric-cards/src/components/MetricCardContainer.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <div class="kong-ui-public-metric-cards">
+    <div
+      v-if="allCardsHaveErrors && errorTitle"
+      class="error"
+    >
+      <KIcon
+        class="error-icon"
+        icon="warning"
+        size="32"
+      />
+      <div class="error-display">
+        <div class="error-display-title">
+          {{ errorTitle }}
+        </div>
+        <div
+          v-if="errorMessage"
+          class="error-display-message"
+        >
+          {{ errorMessage }}
+        </div>
+      </div>
+    </div>
+    <template
+      v-for="(card, index) in cards"
+      v-else
+    >
+      <MetricCardLoadingSkeleton
+        v-if="loading"
+        :key="`skeleton-${index}`"
+        :class="cardSize === MetricCardSize.Small ? 'loading-tabs-small' : 'loading-tabs-large'"
+      />
+      <MetricsCard
+        v-else
+        :key="index"
+        v-bind="formatCardValues(card)"
+        :card-size="cardSize"
+        :error-message="card.errorMessage"
+        :has-error="card.hasError"
+        :title="card.title"
+        :tooltip="card.tooltip"
+      />
+    </template>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed, PropType } from 'vue'
+import approxNum from 'approximate-number'
+import { MetricCardSize, MetricsCardDef } from '../types'
+import { changePolarity, metricChange, defineIcon } from '../utilities'
+import { KIcon } from '@kong/kongponents'
```

Comment:
issue: you shouldn't import Kongponents, they are globally available in the host app. 
```suggestion
```